container/build.py

```diff
@@ -259,16 +262,26 @@ def cloud_build(project,
       full_path = os.path.join(dirpath, f)
       os.chmod(full_path, 0o644)
 
+  # create the required steps for the GCB yaml file as a list of dicts
+  gcb_steps = get_gcb_steps(project, container, tag, package, target,
```

Comment:
I'm not sure about have these additional set of flags to choose whether or not add this or that step. We could list out which steps are needed to be added directly. Sort of moving and unpacking get_gcb_steps() here.